gno.land/pkg/gnoclient/client.go

```diff
@@ -14,15 +13,15 @@ type Client struct {
 // validateSigner checks that the signer is correctly configured.
 func (c Client) validateSigner() error {
 	if c.Signer == nil {
-		return errors.New("missing Signer")
+		return errMissingSigner
```

Comment:
Errors should be always public. We are extracting them as a global variable to make them comparable from the caller, making possible better error handling.